apps/scripts/pipe

```diff
@@ -89,13 +94,13 @@ while [ $# -gt 0 ]; do
                 h)  show_help; exit ;;
                 s)  server=$2; shift ;;
                 u)  user=$2; shift ;;
-		v)  verbose=' -v '; shift ;;
-                d)  opts="-F dryRun=true $opts";;
-                b)  opts="-F bindings='`generate_json "$2"`' $opts"; shift ;;
+                v)  verbose="--verbose"; shift ;;
+                d)  opts="--form dryRun=true $opts";;
+                b)  opts="--form bindings='$(generate_json "$2")' $opts"; shift ;;
```

Comment:
@lukdz I observe that the bindings to the content script are failing with the latest pull. 
For example: 
 ./pipe test.txt -n 1000 -b **modifiedDate=2023-08-30T10:20:20.206Z**
 
 The script generates curl command like this for the binding and the script at AEM is failing with binding parameter not availalble:
 **-form ‘bindings=‘\’‘{“modifiedDate”:“2023-08-30T10:20:20.206Z”}‘\’’'**
 
 cc: @npeltier

Reply:
I tested example providede:
```bash
$ ./pipe test.txt -n 1000 -b modifiedDate=2023-08-30T10:20:20.206Z
+ curl --user admin:admin --show-error --silent --fail --form pipe_cmd=test.txt --form 'bindings='\''{"modifiedDate":"2023-08-30T10:20:20.206Z"}'\''' --form size=1000 http://localhost:4502/apps/dx/scripts/exec.json
```
Reverting:
```diff
-                b)  opts="--form bindings='$(generate_json "$2")' $opts"; shift ;;
+                b)  opts="-F bindings='`generate_json "$2"`' $opts"; shift ;;
```
Doesn't change the result:
```bash
$ ./pipe test.txt -n 1000 -b modifiedDate=2023-08-30T10:20:20.206Z
+ curl --user admin:admin --show-error --silent --fail --form pipe_cmd=test.txt -F 'bindings='\''{"modifiedDate":"2023-08-30T10:20:20.206Z"}'\''' --form size=1000 http://localhost:4502/apps/dx/scripts/exec.json
```
But adding back eval:
```diff
-  curl \
+ eval curl \
```
Does change the result:
```bash
$ ./pipe test.txt -n 1000 -b modifiedDate=2023-08-30T10:20:20.206Z
+ eval curl --user admin:admin --show-error --silent --fail --form pipe_cmd=test.txt -F 'bindings='\''{"modifiedDate":"2023-08-30T10:20:20.206Z"}'\''' --form size=1000 http://localhost:4502/apps/dx/scripts/exec.json
++ curl --user admin:admin --show-error --silent --fail --form pipe_cmd=test.txt -F 'bindings={"modifiedDate":"2023-08-30T10:20:20.206Z"}' --form size=1000 http://localhost:4502/apps/dx/scripts/exec.json
```
Looks like `eval` is removing `'\''`.
The proper fix seems to be removal of single quotes:
```diff
-                b)  opts="--form bindings='$(generate_json "$2")' $opts"; shift ;;
+                b)  opts="--form bindings=$(generate_json "$2") $opts"; shift ;;
```
With results in (without adding back `eval`):
```bash
$ ./pipe test.txt -n 1000 -b modifiedDate=2023-08-30T10:20:20.206Z
+ curl --user admin:admin --show-error --silent --fail --form pipe_cmd=test.txt --form 'bindings={"modifiedDate":"2023-08-30T10:20:20.206Z"}' --form size=1000 http://localhost:4502/apps/dx/scripts/exec.json
```

This fix would require validation from @praveenrv 
afterward, it can be merged: https://github.com/adobe/adobe-dx/pull/202

Reply:
@lukdz thanks for the details and fix. I see it working after removing the single quotes. 

````
opts="--form bindings=$(generate_json "$2") $opts"; shift ;;

test:

./pipe test.txt -n 1000 -b modifiedDate=2023-08-30T10:20:20.206Z
+ curl --user admin:admin --show-error --silent --fail --form pipe_cmdfile=@test.txt --form 'bindings={"modifiedDate":"2023-08-30T10:20:20.206Z"}' --form size=1000 http://localhost:4502/apps/dx/scripts/exec.json

cc: @npeltier